Sources/SwiftDocC/SourceRepository/SourceRepository.swift

```diff
@@ -9,33 +9,40 @@
 */
 
 import Foundation
+import ArgumentParser
 
 /// A remote repository that hosts source code.
 public struct SourceRepository {
     /// The path at which the repository is cloned locally.
     public var checkoutPath: String
```

Comment:
If this was defined as a `URL` then the issue with absolute and relative paths would already be handled by the `URL(fileURLWithPath:)` initializer.